Sources/Bow/Typeclasses/Divisible.swift

```diff
@@ -0,0 +1,12 @@
+/// Divisible extends Divide by providing an empty value.
+public protocol Divisible: Divide {
+    /// Provides an empty value for Kind<Self, A>
+    static func conquer<A>() -> Kind<Self, A>
+}
+
+// MARK: Syntax for Divisible
+public extension Kind where F: Divisible {
+    func conquer() -> Kind<F, A> {
```

Comment:
This method must be static. The empty value is not associated to a specific value in the `F` context, but to the context itself. Look, for instance, at `Monoid` as an example.